interface-definitions/dns-forwarding.xml.in

```diff
@@ -133,8 +133,12 @@
                             <format>@</format>
                             <description>Root record</description>
                           </valueHelp>
+                          <valueHelp>
+                            <format>all</format>
+                            <description>Wildcard record (*.example.com)</description>
```

Comment:
Call me picky, but I think "Wildcard record (any subdomain)" would read better. Mention of a specific domain may mislead someone into thinking it applies only to that domain, even if it's a well-known reserved domain (there are also beginners who don't yet know that it's reserved).

Reply:
Sure it makes sense 
The idea was as we have “reset up bgp all” and don’t have “any” 
:)